knexfile.js

```diff
@@ -0,0 +1,41 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
```

Comment:
As a longer-term tech debt thing, I wonder if we should re-visit this. I think we should be relying on passing these things in as environment variables rather than dictating what client we're using in a config file. i.e. What if I want to use sqlite or mysql during development?